src/main/resources/db/changelog/changes/02-add-test-book.yaml

```diff
@@ -0,0 +1,29 @@
+databaseChangeLog:
+  - changeSet:
+      id: insertBooks
```

Comment:
Do you really have to create a file to add a book to database?What if you have to add 10 000 books in your DB?

Reply:
Then I write method that add books. It is just test that defines liquibase work correct